.github/workflows/build.yaml

```diff
@@ -121,3 +121,46 @@ jobs:
 
       - name: Run phpunit/phpunit
         run: vendor/bin/phpunit
+
+  code-coverage:
+    name: Code Coverage
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: xdebug
```

Comment:
Installing `xdebug` as code coverage driver here.

💁‍♂️ For reference, see:

- https://github.com/shivammathur/setup-php#xdebug

Reply:
A-ha! 👍